seed/challenges/02-data-visualization-certification/sass.json

```diff
@@ -1,19 +1,307 @@
 {
   "name": "Sass",
   "order": 13,
-  "isComingSoon": true,
-  "time": "5 hours",
+  "isComingSoon": "true",
+  "time": "5h",
   "helpRoom": "HelpDataViz",
   "challenges": [
     {
-      "id": "bd7158d8c423ede2aeb5bdee",
-      "title": "Learn Sass Challenges",
+      "id": "5690a4625b3908042c47b327",
+      "title": "CSS with Superpowers",
+      "description": [
+        "Q: How can we write better CSS?",
+        "A: Use a preprocessor.",
+        "",
+        "What is a CSS preprocessor?",
+        "CSS preprocessors take code written in the preprocessed language and then convert that code into CSS. Pretty basic, right?",
+        "",
+        "So what's the point of using one?",
+        "CSS preprocessors such as <code>Sass</code> or <code>Less</code> allow you to do nifty things that regular CSS can't do. They allow you to create <code>variables</code>, write more compact code with <code>nesting</code>, <code>import</code> other CSS files and much more.",
+        "",
+        "In this series of Waypoints, we will be teaching you the basics of <code>Sass</code>"
+      ],
+      "challengeSeed": [],
+      "tests": [],
+      "type": "waypoint",
+      "challengeType": 0
+    },
+    {
+      "id": "5690ad185b3908042c47b328",
+      "title": "Storing Data with Sass Variables",
+      "description": [
+        "Just like in Javascript, you can declare variables to store data in. ",
+        "In Javascript, you would use the <code>var</code> keyword to define a variable. When decaring variables in <code>Sass</code>, you start by using a <code>$</code> followed by a string.",
+        "Here is an example: <code>$color = green;</code>",
+        "Making variables become exteremly useful when we want similar elemnts to have the same color, for example. If wanted to change the color to maybe a slightly different shade, all we would have to do is change the variables value.",
```

Comment:
L32: Remove space at the end
L33: "Javascript" => "JavaScript" and "When decaring variables..." => "When **declaring** variables..."
L35:
- "...when we want similar elemnts..." => "...when we want similar **elements**..."
-  "If wanted..." => "If **you** wanted..."
- Remove the "maybe" in "maybe a slightly different shade.." to be "change the color to a slightly different..."
- "...change the variables value." => "...change the variable's value."